pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -165,9 +165,14 @@ def fractional_octave_bands(
         num_fractions,
         sampling_rate=None,
         frequency_range=(20.0, 20e3),
-        order=14):
+        order=14,
+        check_tolerance_class=1):
     """Create and/or apply an energy preserving fractional octave filter bank.
 
+    The filters are in accordance with IEC 61260-1:2014 [#]_ for octave-band
+    filters of order 2 and higher, and for one-third-third-octave-band filters
+    of order 6 and higher (see examples below).
```

Comment:
```suggestion
    The filters are in accordance with IEC 61260-1:2014 [#]_ for octave-band
    filters of order 2 and higher, and for one-third-octave-band filters
    of order 6 and higher (see examples below), otherwise a warning is given. 
```

i would also wote for not input class parameter, we are